PelotonEppSdk/Models/EventCustomField.cs

```diff
@@ -10,18 +11,23 @@ public class EventCustomField
         /// <summary>
         /// The name of this custom field.
         /// </summary>
+        [StringLength(50, MinimumLength = 0, ErrorMessage = "Name must be 50 or fewer characters long.")]
```

Comment:
Agreed. Changed all occurrences of "... characters long" to "... characters in length"